FiducialRegistrationWizard/MRML/vtkMRMLFiducialRegistrationWizardNode.cxx

```diff
@@ -319,6 +320,56 @@ int vtkMRMLFiducialRegistrationWizardNode::UpdateModeFromString( std::string upd
 }
 
 //------------------------------------------------------------------------------
+void vtkMRMLFiducialRegistrationWizardNode::SetInputFormat( int newInputFormat )
+{
+  if ( this->GetInputFormat() == newInputFormat )
+  {
+    // no change
+    return;
+  }
+  this->InputFormat = newInputFormat;
+  this->Modified();
+  this->InvokeCustomModifiedEvent(InputDataModifiedEvent);
+}
+
+//------------------------------------------------------------------------------
+std::string vtkMRMLFiducialRegistrationWizardNode::InputFormatAsString( int inputFormat )
+{
+  switch ( inputFormat )
+  {
+    case INPUT_FORMAT_ORDERED_PAIRS:
+    {
+      return "Ordered Pairs";
```

Comment:
This string should be for machine reading (to write to XML). The best is to store strings that will be displayed to users in the .ui file. If you store the displayable string in the source file then use the tr() macro to allow translation.